jsonschema-validation.xml

```diff
@@ -491,6 +491,18 @@
                     </t>
                 </section>
 
+                <section title="requiredAllExcept">
+                    <t>
+                        The value of this keyword MUST be an array.
+                        Elements of this array, if any, MUST be strings, and MUST be unique.
+                    </t>
+                    <t>
+                        An object instance is valid against this keyword if every property name
+                        declared in "properties" except those that appear in the array is the name
+                        of a property in the instance.
```

Comment:
This wording could be improved:

An object instance is valid against this keyword if every property name
declared in "properties" is the name of a property in the instance, with
the exception of those that appear in this keyword's array.

Reply:
Perhaps we could also mention one usecase that could become a common shortcut: `"requiredAllExcept": []` indicates that all properties in `properties` are required.

Reply:
Yeah, that wording sounds better. I'll update it.

I'd rather not include usage patterns in the spec. I don't think we do that anywhere else, so it would be a little out of place. It should definitely be mentioned in UJS tho.